README.md

```diff
@@ -1,2 +1,88 @@
-# DumE
-Open Projects 2021
+# DumE
+Open Projects 2021
+***
+
+<p align="center">
+  <img width="560" height="400" src="https://github.com/rodion0917/Random/blob/main/Dum-E%20in%20Action%20-%20Green%20Object.png">
+  
+  <i>DumE</i>
+</p>
+
+***
+
+## Abstract
+Dum-E is a pick-up and place, robotic arm. Equipped with image processing and computer vision,it can 
+detect and classify objects on the basis of their color or shape. Currently, it can operate in static
+environments, that is where obstacles and the object are not moving.
+***
+
+## Motivation
+The name Dum-E, itself is a hint of our source if inspiration for doing this project. After all,who 
+hasn't watched Iron Man. Dum-E first appeared in Iron Man I, though it keep messing things for Tony, 
+it had saved his life too. 
+Our inspiration was simply to make something cool and learn the basics of robotics, computer vision,
+image processing and the various platforms which can be used for designing and simulating our robotic arm. 
+***
+
+## Workflow
+<p align="center">
+  <img width="700" height="600" src="https://github.com/rodion0917/Random/blob/main/Work-flow%20chart.png" "Workflow">
+</p>
+<i>Workflow</i>
+
+<p align="center">
+  <img width="850" height="200" src="https://github.com/rodion0917/Random/blob/main/Picking%20up%20the%20object%20-%20Workflow.png">
+</p>
+<i>Picking up the object</i>
+
+<p align="center">
+  <img width="560" height="200" src="https://github.com/rodion0917/Random/blob/main/Placing%20the%20object%20-%20Workflow.png">
+</p>
+<i>Placiing the object</i>
+
+***
+

```

Comment:
Add a section named 'Software Aspect' after 'Workflow'. And hyperlink a README file by creating it inside src folder where you'll explain the code and algorithm. Refer this: https://github.com/marsiitr/RoverSim#software-aspects